scripts/process/scaleStates.R

```diff
@@ -0,0 +1,75 @@
+#scale states according to water use
+# two different sized states with equal water use should have the 
+# same area
+library(rgeos)
+library(dplyr)
+
+process.scaleStates <- function(viz){
+  #could put these in yaml too?
+  coreCats <- c("Thermoelectric", "Irrigation", "Public Supply", "Industrial")
+  
+  #read sp, wuClean
+  statePoly <- readData(viz[['depends']]$stateMap)
```

Comment:
Note I changed this into a list, so you need `statePoly <- readData(viz[['depends']]$stateMap)$states` here if you pull upstream